src/deps/bulkGetShim.js

```diff
@@ -77,7 +77,7 @@ function bulkGet(db, opts, callback) {
     }
 
     // globally-supplied options
-    ['revs', 'attachments', 'binary'].forEach(function (param) {
+    ['revs', 'attachments', 'binary', 'timeout', 'cache'].forEach(function (param) {
```

Comment:
This fix doesn't work, but neither does including `ajax` in this list.  What _does_ work is adding a line which sets `docOpts = { ajax: { timeout: 12345 } };`.